src/main/webapp/WEB-INF/static/js/import-crosses.js

```diff
@@ -694,16 +689,22 @@ var ImportCrosses = {
 	isCrossImportSettingsValid: function(importSettings) {
 		'use strict';
 		var valid = true;
-		if (!importSettings.crossNameSetting.prefix || importSettings.crossNameSetting.prefix === '') {
+		if($('#harvestMonthDropdown').val() === '') {
```

Comment:
Will it yield the same result if we use i`mportSettings.additionalDetailsSetting.harvestDate === ''`? 
If so, better to use that for code consistency